### DISCUSSION TITLE
[Question]: would you recommend the reolink RLC-823A and/or trackmix  for frigate?

### DISCUSSION THREAD

**Discussion by gurkburk76** [blakeblackshear/frigate#10851] [Ask A Question]
### What is your question:

So i might get of of these soon if they work well with frigate, i don't think that the autotracking works from frigate but if the cameras autotracking works well enought that's fine. I really only need it to track when stuff comes into it's FOV and reset to a given position after it looses it's target.
Seems like i can move ptz from frigate from what i've read on the docs so that's fine.

Anyone that have this camera and could tell me of stuff that bother them or other quirks one should know about?
It's in a good pricerange but perhaps there are alternatives that also are within that pricepoint but are a much better choice, feel free to give me ideas :)

**Comment by Sch0field9** [comment]
No, I dont recommend it.
I have two 823A-16X. Their ONVIF protocol is wonky. And frigate doesn't really like it if the camera is moving itself instead of controlled by frigate. I think you are better of with a PTZ camera that has the ONVIF-T protocol correctly implemented.
I also have problems with the 2-way-audio. I havent figured it out where the problem is. Could be that its a network or a HomeAssistant issue.

**Reply by hyacin75** [reply]
> I think you are better of with a PTZ camera that has the ONVIF-T protocol correctly implemented.

Any specific suggestions?

I just bought a brand new RLC-823S1 specifically to use with Frigate and am sick to my stomach and almost in tears that the autotracking won't work, which is literally why I bought a new camera at all, and specifically this one.

I'm sure Frigate will be confused as hell if I just let the camera do the autotracking itself.  :'(

**Reply by robcoward** [reply]
I have a RLC-823S1 that I have configured with Frigate. The RTSP stream is working fine via go2rtc. I have also added the onvif config to the camera and it brings up the PTZ controls along with the preset positions I configured on the camera and allows manual moving and zooming of the camera, and the camera's built-in tracking works just fine. What I havent managed to get working yet is the frigate controlled tracking, or automatic zoom. It seems like others have suggested, the reolink onvif responses are not quite matching the standard that frigate is expecting.
